services/core-web/common/constants/API.js

```diff
@@ -106,6 +106,18 @@ export const VARIANCE_DOCUMENT = (mineGuid, varianceGuid, documentManagerGuid) =
 export const VARIANCE_STATUS_CODES = "/variances/status-codes";
 export const VARIANCE_DOCUMENT_CATEGORY_OPTIONS = "/variances/document-categories";
 
+// Project Summaries
+export const MINE_PROJECT_SUMMARIES = (mineGuid) => `/mines/${mineGuid}/project-summaries`;
+export const PROJECT_SUMMARIES = () => "/project-summaries";
+export const PROJECT_SUMMARY = (mineGuid, projectSummaryGuid) =>
+  `/mines/${mineGuid}/project-summaries/${projectSummaryGuid}`;
+export const PROJECT_SUMMARY_DOCUMENTS = (mineGuid, projectSummaryGuid) =>
+  `/mines/${mineGuid}/project-summaries/${projectSummaryGuid}/documents`;
+export const PROJECT_SUMMARY_DOCUMENT = (mineGuid, projectSummaryGuid, documentManagerGuid) =>
+  `/mines/${mineGuid}/project-summaries/${projectSummaryGuid}/documents/${documentManagerGuid}`;
+export const PROJECT_SUMMARY_STATUS_CODES = "/mines/project-summary-status-codes";
+export const PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS = "/mines/project-summary-document-types";
+
```

Comment:
Are PROJECT_SUMMARY_STATUS_CODES && PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS being used? I see they were added to the static content reducer/state so I would assume not.

Reply:
I added them in anticipation of having dropdowns, but no currently they aren't being used.

Reply:
The dropdowns will come from the staticContent selectors, so they should be covered via a different path (which you have already added) these shouldn't be needed